conf/defaults.yaml

```diff
@@ -63,7 +63,7 @@ nimbus.supervisor.timeout.secs: 60
 nimbus.monitor.freq.secs: 10
 nimbus.cleanup.inbox.freq.secs: 600
 nimbus.inbox.jar.expiration.secs: 3600
-nimbus.code.sync.freq.secs: 300
+nimbus.code.sync.freq.secs: 120
```

Comment:
5 min to sync blobs seems to be pretty large, hopefully 2 min should be good enough since we also have callback enabled